src/scales/scale.time.js

```diff
@@ -211,13 +213,14 @@ function toTimestamp(input, options) {
 	return value;
 }
 
-function parse(input, scale) {
+function parse(scale, input) {
 	if (helpers.isNullOrUndef(input)) {
 		return null;
 	}
 
+	var adapter = scale._adapter;
```

Comment:
adapter only used once, so better not to alias

Reply:
done